core/bin/zksync_api/src/bin/providers/proxy_utils.rs

```diff
@@ -0,0 +1,138 @@
+use std::{
+    collections::HashMap,
+    sync::Arc,
+    time::{Duration, Instant},
+};
+use tokio::sync::Mutex;
+
+use actix_web::{web::Json, HttpResponse};
+use serde_json::{json, Value};
+use zksync_config::DevTickerConfig;
+
+pub(crate) const API_URL: &str = "https://api.coingecko.com"; // TODO: could be eventually added to the config files
+pub(crate) const API_PATH: &str = "/api/v3"; // TODO: could be eventually added to the config files
```

Comment:
I generally don't like that because when you change the line you'll forget to change the comment above. but if the comment is on the same line you're less likely to forget. especially if you're just adding something somewhere that has not got anything to do with the commented-on lines (such as when you rebase code).

Reply:
Although inline comment is allowed, It looks it's  a de-facto standard writing comments before the line they refer to. Please refer to:
- https://softwareengineering.stackexchange.com/questions/126601/comment-before-or-after-the-relevant-code
- https://queue.acm.org/detail.cfm?id=1053354
- [not very related, since it's specific for Joomla](https://developer.joomla.org/coding-standards/inline-code-comments.html#:~:text=Comments%20should%20precede%20the%20code,be%20on%20their%20own%20lines.)